spec/spec.yaml

```
@@ -3936,6 +3952,180 @@ paths:
           schema:
             $ref: "./definitions.yaml#/definitions/V4GenericResponse"
 
+  /v5/clusters/{cluster_id}/labels/:
+    get:
+      operationId: getClusterLabels
+      tags:
+        - cluster labels
+      summary: Get cluster labels
+      description: |
+        Return labels attached to the supplied cluster.
```

```suggestion
        Returns the labels that this cluster is labelled with
```

`attached` is kind of a new domain word, I don't think we need to introduce that. When we want to add a label to a cluster, we set the labels (since thats what we're calling it in the put action), and the cluster is labelled.